### DISCUSSION TITLE
"I made this for you" [Sprites/Clothes]

### DISCUSSION THREAD

**Discussion by Belial-alt** [Monika-After-Story/MonikaModDev#7935] [QA - code and content creation (technical)] [answered]
This is my first time posting on github, so sorry if I don't explain this very well. 
I am figuring how to make my own clothes for Monika and I thought it would be interesting if there could be a code implication to say that you made the sprite yourself? 
I only know some very basic Java and C#, but maybe something with writing the JSON being able to write something like `handmade="true";` and MAS can recognize that for a scripted event? (Like a pop up saying you made it yourself?)
I'm sorry if this makes no sense or would be impossible lmao, but I thought it would be interesting for some-sort of scripted event when you make Monika gifts :)

**Comment by Booplicate** [comment, marked as answer]
You could specify your own reaction label for the spritepack where you could mention that.
